cyber_dependabot.py

```diff
@@ -0,0 +1,80 @@
+import os
+import datetime
+import logging
+from multiprocessing import Pool
+from collections import Counter
+from typing import Iterator, Callable
+
+from addict import Dict
+import requests
+
+import storage
+import config
+
+
+def pmap(f: Callable, collection: Iterator, size: int = 10) -> list:
+    """
+    Applies `f` in parallel over `collection`.
+    Pool `size` has a sensible default of 10.
+    """
+    with Pool(size) as p:
+        return p.map(f, collection)
+
+
+def put(path: str) -> requests.models.Response:
+    """
+    Adapter of `requests.put()`, supplying github credentials.
+    """
+    ROOT_URL = "https://api.github.com"
+    headers = {
+        "Authorization": f'token {os.environ.get("TOKEN", "")}',
```

Comment:
we're sending the github token to the lambda as an encrypted kms token that will need to be decrypted in the python function itself - might be worth putting this in now?
i.e.:
ENCRYPTED_GITHUB_TOKEN = os.environ.get["GITHUB_TOKEN"]
DECRYPTED_GITHUB_TOKEN = boto3.client('kms').decrypt(CiphertextBlob=b64decode(ENCRYPTED_GITHUB_TOKEN))['Plaintext']